src/Microsoft.DotNet.Build.Tasks.Feed/PushToBlobFeed.cs

```diff
@@ -51,13 +63,22 @@ public override bool Execute()
             return !Log.HasLoggedErrors;
         }
 
-        private List<string> ConvertToStringLists(ITaskItem[] taskItems)
+        private List<string> ConvertToStringLists(ITaskItem[] taskItems, bool isNugetPackage)
```

Comment:
I would change isNugetPackage to fileextension to make this method a little more general.